src/measure_theory/integral/circle_integral.lean

```diff
@@ -164,6 +175,15 @@ lemma lipschitz_with_circle_map (c : ℂ) (R : ℝ) :
 lipschitz_with_of_nnnorm_deriv_le (differentiable_circle_map _ _) $ λ θ,
   nnreal.coe_le_coe.1 $ by simp
 
+lemma circle_map_inv_continuous_on (R : ℝ) (hR : 0 < R) (z w : ℂ) (hw : w ∈ ball z R) :
+ continuous_on (λ θ, (circle_map z R θ - w)⁻¹) [0, 2*π] :=
+begin
+  simp_rw ←one_div,
+  apply_rules [continuous_on.div, continuous_const.continuous_on, continuous_on.sub,
+    (continuous_circle_map z R).continuous_on, continuous_const.continuous_on],
+  refine (λ _ _,  (circle_map_ne_on_ball hR hw) _),
```

Comment:
Isn't it more idiomatic to use "exact" rather than "refine" when lean can fill in all the blanks for you?

Reply:
Hmm yes I suppose you're right. I don't know if in this case there are any other reasons to choose between `refine` and `exact`, so I'm happy to change it.